src/app.rs

```diff
@@ -207,6 +207,12 @@ pub fn build_app() -> App<'static, 'static> {
                 .hidden_short_help(true),
         )
         .arg(arg("pattern"))
+        .arg(
+            arg("path-separator")
```

Comment:
we should probably also add `.hidden_short_help(true)`. I don't think it is important enough to be shown in the concise overview (`fd -h`).